packages/backend/src/open_payments/quote/service.test.ts

```diff
@@ -506,49 +473,55 @@ describe('QuoteService', (): void => {
 
       test.each`
         incomingAmountValue | fixedFee | basisPointFee | expectedQuoteDebitAmountValue | description
-        ${3364n}            | ${0}     | ${0}          | ${3365n}                      | ${'no fees'}
-        ${3364n}            | ${150}   | ${0}          | ${3515n}                      | ${'fixed fee'}
-        ${3364n}            | ${0}     | ${200}        | ${3432n}                      | ${'basis point fee'}
-        ${3364n}            | ${150}   | ${200}        | ${3582n}                      | ${'fixed and basis point fee'}
+        ${1000}             | ${0}     | ${0}          | ${1000n}                      | ${'no fees'}
+        ${1000n}            | ${150}   | ${0}          | ${1150n}                      | ${'fixed fee'}
+        ${1000n}            | ${0}     | ${200}        | ${1020n}                      | ${'basis point fee'}
+        ${1000n}            | ${150}   | ${200}        | ${1170n}                      | ${'fixed and basis point fee'}
       `(
         '$description',
-        withConfigOverride(
-          () => config,
-          { slippage: 0 },
-          async ({
-            incomingAmountValue,
-            fixedFee,
-            basisPointFee,
-            expectedQuoteDebitAmountValue
-          }): Promise<void> => {
-            const incomingPayment = await createIncomingPayment(deps, {
-              paymentPointerId: receivingPaymentPointer.id,
-              incomingAmount: {
-                assetCode: asset.code,
-                assetScale: asset.scale,
-                value: incomingAmountValue
-              }
-            })
-            await Fee.query().insertAndFetch({
-              assetId: asset.id,
-              type: FeeType.Sending,
-              fixedFee,
-              basisPointFee
-            })
-
-            const quote = await quoteService.create({
-              paymentPointerId: sendingPaymentPointer.id,
-              receiver: incomingPayment.getUrl(receivingPaymentPointer)
-            })
-            assert.ok(!isQuoteError(quote))
-
-            expect(quote.debitAmount).toEqual({
+        async ({
+          incomingAmountValue,
+          fixedFee,
+          basisPointFee,
+          expectedQuoteDebitAmountValue
+        }): Promise<void> => {
```

Comment:
Can you elaborate on how there isnt an off by one issue here? Did we gain any insight into that issue?

As I recall the default slippage introduced some variance in this test and manual tests (via postman), which is why I tried to test without slippage originally. Although it appears that using the default slippage doesnt modify the values at all here?

Reply:
aha, makes sense. Good, glad there is something preserving that issue.